src/screen/splash/mod.rs

```diff
@@ -42,9 +46,12 @@ fn spawn_splash(mut commands: Commands, asset_server: Res<AssetServer>) {
                         width: Val::Percent(70.0),
                         ..default()
                     },
-                    image: UiImage::new(
-                        asset_server.load("embedded://bevy_template/screen/splash/splash.png"),
-                    ),
+                    image: UiImage::new(asset_server.load_with_settings(
+                        "embedded://bevy_template/screen/splash/splash.png",
+                        |settings: &mut ImageLoaderSettings| {
+                            settings.sampler = ImageSampler::linear();
```

Comment:
```suggestion
                            // This is already the default, but we are setting it explicitly
                            // because you may want to create a pixel-art game using
                            // `ImagePlugin::default_nearest()`, which would break this image
                            settings.sampler = ImageSampler::linear();
```